src/Infrastructure/Service/ValueReceiver/ApiInteractionProcessor.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Service\ValueReceiver;
+
+use SprykerSdk\Sdk\Core\Application\Dependency\InteractionProcessorInterface;
+use SprykerSdk\Sdk\Core\Application\Dto\ReceiverValueInterface;
+use SprykerSdk\Sdk\Infrastructure\Event\RequestDataInjectorInterface;
+use SprykerSdk\Sdk\Infrastructure\Exception\InvalidRequestDataException;
+use SprykerSdk\SdkContracts\Enum\ValueTypeEnum;
+
+class ApiInteractionProcessor implements InteractionProcessorInterface, RequestDataInjectorInterface
+{
+    /**
+     * @var array
+     */
+    protected array $data;
+
+    /**
+     * @param array $data
+     *
+     * @return void
+     */
+    public function setRequestData(array $data): void
+    {
+        $this->data = $data;
+    }
+
+    /**
+     * @param string $key
+     *
+     * @return bool
+     */
+    public function has(string $key): bool
+    {
+        return !empty($this->data[$key]);
```

Comment:
1. it can be replaced by `get` method completely 
2. if it is necessary to keep it, please rename it to `hasRequestItem`